### PR TITLE
Update device-rename.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that the change in device name will only change in Intune and not on the device (Android Enterprise devices)

Per issue#https://github.com/MicrosoftDocs/memdocs/issues/3009

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/remote-actions/device-rename.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ If a device isn't listed here, it's not supported.
 
 This feature doesn't support renaming hybrid Azure AD Windows devices.
 
+> [!NOTE]
+> Rename of Android Enterprise devices will only change the name on Intune and not on the device itself. The Device name in Intune is a friendly name that the users are free to change.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/device-rename.md:51]:
```suggestion
> Renaming of Android Enterprise devices will only change the name in Intune and not on the device itself. The Device name in Intune is a friendly name that users are free to change.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Smritib17 Please review and Signoff this PR. Thanks

**Review by Smritib17 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**